### ISSUE TITLE
Can't build demo

### ISSUE THREAD

**Issue by 3rror404** [peoplewareDo/nativescript-svg#8]
I don't seem to be able to build the demo app on Mac. Running the following in the demo folder results in a build error:

```
$ tns build ios
```

or

```
$ tns run ios
```

iOS
```
The following build commands failed:
	CompileC /Users/daniel_rochelle/Library/Developer/Xcode/DerivedData/demo-gysnbxxljfxegdfoslhyhbuzhvly/Build/Intermediates/Pods.build/Debug-iphonesimulator/SVGKit.build/Objects-normal/i386/CAShapeLayerWithHitTest.o SVGKit/Source/QuartzCore\ additions/CAShapeLayerWithHitTest.m normal i386 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

Android:

```
* Where:
Build file '/Users/daniel_rochelle/Documents/GitHub/repos/nativescript-svg/demo/platforms/android/build.gradle' line: 705

* What went wrong:
Execution failed for task ':myMergeDex'.
> Multiple dex files define Lcom/tns/NativeScriptApplication;
```

I've also tried `$ npm install` before building with the same result.

Any ideas? Thanks

```
OSX: 10.11.6
XCode: 8.2.1
┌──────────────────┬─────────────────┬────────────────┬──────────────────┐
│ Component        │ Current version │ Latest version │ Information      │
│ nativescript     │ 2.4.2           │ 2.4.2          │ Up to date       │
│ tns-core-modules │ 2.0.1           │ 2.4.4          │ Update available │
│ tns-android      │ 2.0.0           │ 2.4.1          │ Update available │
│ tns-ios          │ 2.4.0           │ 2.4.0          │ Up to date       │
└──────────────────┴─────────────────┴────────────────┴──────────────────┘
```
---
After further testing it seems I am also unable to build a basic app using the official [NativeScript Hello World Template](https://github.com/NativeScript/template-hello-world) after installing this plugin.

Same error as above.

**Comment by victorsosa** [comment]
Fix for the demo is comming on 1.1.4 soon

**Comment by victorsosa** [comment]
fixed
